mbed_lstools/platform_database.py

```diff
@@ -42,228 +42,248 @@
+        u'1237': u'UBLOX_EVK_NINA_B1',
+        u'1300': u'NUC472-NUTINY',
+        u'1301': u'NUMBED',
+        u'1302': u'NUMAKER_PFM_NUC472',
+        u'1303': u'NUMAKER_PFM_M453',
+        u'1304': u'NUMAKER_PFM_M487',
+        u'1305': u'NUMAKER_PFM_M2351',
+        u'1306': u'NUMAKER_PFM_NANO130',
+        u'1307': u'NUMAKER_PFM_NUC240',
+        u'1549': u'LPC1549',
+        u'1600': u'LPC4330_M4',
+        u'1605': u'LPC4330_M4',
+        u'2000': u'EFM32_G8XX_STK',
+        u'2005': u'EFM32HG_STK3400',
+        u'2010': u'EFM32WG_STK3800',
+        u'2015': u'EFM32GG_STK3700',
+        u'2020': u'EFM32LG_STK3600',
+        u'2025': u'EFM32TG_STK3300',
+        u'2030': u'EFM32ZG_STK3200',
+        u'2035': u'EFM32PG_STK3401',
+        u'2040': u'EFM32PG12_STK3402',
+        u'2041': u'TB_SENSE_12',
+        u'2045': u'TB_SENSE_1',
+        u'2100': u'XBED_LPC1768',
+        u'2201': u'WIZWIKI_W7500',
+        u'2202': u'WIZWIKI_W7500ECO',
+        u'2203': u'WIZWIKI_W7500P',
+        u'2500': u'ADV_WISE_1570',
+        u'3001': u'LPC11U24',
+        u'4000': u'LPC11U35_Y5_MBUG',
+        u'4005': u'NRF51822_Y5_MBUG',
+        u'4100': u'MOTE_L152RC',
+        u'4337': u'LPC4337',
+        u'4500': u'DELTA_DFCM_NNN40',
+        u'4501': u'DELTA_DFBM_NQ620',
+        u'4502': u'DELTA_DFCM_NNN50',
+        u'4600': u'REALTEK_RTL8195AM',
+        u'5000': u'ARM_MPS2',
+        u'5001': u'ARM_MPS2_M0',
+        u'5002': u'ARM_BEETLE_SOC',
+        u'5003': u'ARM_MPS2_M0P',
+        u'5005': u'ARM_MPS2_M0DS',
+        u'5007': u'ARM_MPS2_M1',
+        u'5009': u'ARM_MPS2_M3',
+        u'5011': u'ARM_MPS2_M4',
+        u'5015': u'ARM_MPS2_M7',
+        u'5020': u'HOME_GATEWAY_6LOWPAN',
+        u'5500': u'RZ_A1H',
+        u'5501': u'GR_LYCHEE',
+        u'6660': u'NZ32_SC151',
+        u'7010': u'BLUENINJA_CDP_TZ01B',
+        u'7011': u'TMPM066',
+        u'7013': u'TMPM46B',
+        u'7402': u'MBED_BR_HAT',
+        u'7778': u'TEENSY3_1',
+        u'8001': u'UNO_91H',
+        u'9001': u'LPC1347',
+        u'9002': u'LPC11U24',
+        u'9003': u'LPC1347',
+        u'9004': u'ARCH_PRO',
+        u'9006': u'LPC11U24',
+        u'9007': u'LPC11U35_501',
+        u'9008': u'XADOW_M0',
+        u'9009': u'ARCH_BLE',
+        u'9010': u'ARCH_GPRS',
+        u'9011': u'ARCH_MAX',
+        u'9012': u'SEEED_TINY_BLE',
+        u'9900': u'NRF51_MICROBIT',
+        u'C002': u'VK_RZ_A1H',
+        u'C005': u'MTM_MTCONNECT04S',
+        u'C006': u'VBLUNO51',
+        u'C008': u'SAKURAIO_EVB_01',
+        u'C030': u'UBLOX_C030_U201',
+        u'C031': u'UBLOX_C030_N211',
+        u'C032': u'UBLOX_C030_R404M',
+        u'C033': u'UBLOX_C030_R410M',
+        u'C034': u'UBLOX_C030_S200',
+        u'C035': u'UBLOX_C030_R3121',
+        u'FFFF': u'K20 BOOTLOADER',
+        u'RIOT': u'RIOT',
+    },
+    u'jlink': {
+        u'X349858SLYN': {
+            u'platform_name': u'NRF52_DK',
+            u'jlink_device_name': u'nRF52832_xxaa'
```

Comment:
Why do we need this data? Is it important to keep?

Reply:
This is needed for the reason specified in this comment: https://github.com/ARMmbed/mbed-ls/pull/299#issuecomment-363059366